.travis.yml

```diff
@@ -1,39 +1,45 @@
+language: python
 sudo: false
-cache: pip
+cache:
+  apt: true
+  directories:
+    - $HOME/.cache/pip
+
+python:
+  - "2.7"
```

Comment:
Okay this is in the template. I guess we just want to change randomly the order sometimes...